requirements/test-requirements.txt

```diff
@@ -44,7 +44,9 @@ databricks-sdk
 openai<1.0
 # Required for showing pytest stats
 psutil
+# Required for the GraphQL handler
+graphene<4
```

Comment:
I think `graphene` can be removed because it should be installed together with mlflow.